src/pages/community.astro

```diff
@@ -151,25 +159,48 @@ import nixosFoundationLogo from "../assets/image/nixos-foundation-logo.svg";
       The Nix community is spread across various platforms. Here are the
       official and unofficial spaces where you can find help, discuss
       development, and chat with other users.
-    <h3 class="text-3xl font-bold font-heading text-nix-blue mt-4">Official spaces</h3>
-    <p class="text-gray-700">
-      These spaces are monitored and moderated by the NixOS Moderation Team.
-        
-      <ul class="grid grid-cols-1 md:grid-cols-3 gap-4 mt-4 flex-wrap">
-        {
-          platformsMain.map((platform) => (
-            <OfficialCommunityLink
-              href={platform.href}
-              name={platform.name}
-              iconName={platform.iconName}
-              description={platform.description} />
-          ))
-        }
-      </ul>    
-    </p>
+    <div class="grid md:grid-cols-3 gap-4">
+      <div class="md:col-span-2">
+        <h3 class="text-3xl font-bold font-heading text-nix-blue mt-4">Official spaces</h3>
+        <p class="text-gray-700">
+          These spaces are monitored and moderated by the NixOS Moderation Team. This means they are moderated in such a way that they follow the standards and moral values of our community. You can find out more about our moderaiton team <a href="http://localhost:4321/community/teams/moderation">here</a>.
+            
+          <ul class="grid grid-cols-1 md:grid-cols-2 gap-4 mt-4 flex-wrap">
+            {
+              platformsMain.map((platform, i) => (
+                <OfficialCommunityLink
+                  href={platform.href}
+                  name={platform.name}
+                  iconName={platform.iconName}
+                  description={platform.description}
+                  class={i === 0 && "md:col-span-2"}
+                />
+              ))
+            }
+          </ul>    
+        </p>
+      </div>
+      <div class="col-span-1">
+        <h3 class="text-3xl font-bold font-heading text-nix-blue mt-4">Official social-media</h3>
+        <p class="text-gray-700">
+          These spaces are maintained by our NixOS Marketing Team though not monitored and not moderated by the NixOS Moderation Team.
```

Comment:
The hashtags are certainly not maintained by the marketing team. I only subscribe to the mastodon feed, but it is 99% an enduser feed without any kind of maintenance.

Reply:
yeah, kind of forgot to change this, unfortunately it is merged now, will open a separate PR